lib/services/stash-before.js

```diff
@@ -24,12 +23,10 @@ module.exports = function (prop) {
     };
 
     params.query = params.query || {};
-    params.query.$disableStashBefore = true;
+    params.$disableStashBefore = true;
 
     return context.service.get(context.id, params)
       .then(data => {
-        delete params.query.$disableStashBefore;
```

Comment:
@daffl what are your thoughts about leaving the param alone?  We needed to delete this when it was part of `params.query`, but I'm leaning towards leaving this here.  Any downsides you can think of?

Reply:
Yes, we can leave it. Also, I would just use `params.disableStashBefore` as the name, I don't think it needs the `$`.